gateway-ha/src/main/java/io/trino/gateway/ha/resource/GatewayResource.java

```diff
@@ -86,7 +86,7 @@ public Response activateBackend(@PathParam("name") String name)
             this.gatewayBackendManager.activateBackend(name);
         }
         catch (Exception e) {
-            log.error(e);
+            log.error(e, e.getMessage());
```

Comment:
This change is unrelated to "Modify HaGatewayManager to support informative logging when (de)activating backend clusters". Please separate a commit.